mkdocs.yml

```diff
@@ -0,0 +1,77 @@
+site_name: jte documentation
+site_description: "Documentation for jte (Java Template Engine)"
+repo_url: https://github.com/casid/jte/
```

Comment:
This will become the official jte website, not just the documentation.

I'd suggest to change this to 

```
site_name: jte
site_description: "Secure and speedy templates for Java and Kotlin"
```

Reply:
See 6d56a66.